test/cli/test_cli_flow.sh

```diff
@@ -39,6 +39,9 @@ function main {
         post_install_tests
     fi
     noobaa_uninstall
+    noobaa_install_external
+    noobaa_uninstall
+    delete_external_postgres
```

Comment:
maybe add a new test file, or add it as a different flow?
I feel that this will excceed the time we can run the GH actions, or prolong it too mach.
WDYT?

Reply:
moved to random - hope that will make it short enough